src/armTemplates/resources/appServicePlan.ts

```diff
@@ -24,6 +25,10 @@ export class AppServicePlanResource implements ArmResourceTemplateGenerator {
         defaultValue: "",
         type: ArmParamType.String
       },
+      kind: {
```

Comment:
why the name `kind`?

Reply:
That's just the ARM specification required. See [the docs](https://docs.microsoft.com/en-us/azure/azure-functions/functions-infrastructure-as-code) for more details.